getpaid/utils.py

```diff
@@ -58,9 +60,8 @@ def get_backend_settings(backend):
         return {}
 
 
-def build_absolute_uri_for_site(site, view_name, scheme='https',
+def build_absolute_uri_for_site(domain, view_name, scheme='https',
```

Comment:
Can you simply rename this function, `build_absolute_uri` should be just fine, right?